tests/reporter/kubernetes.go

```diff
@@ -1070,7 +1070,6 @@ func (r *KubernetesReporter) executeNodeCommands(virtCli kubecli.KubevirtClient,
 		{command: networkPrefix + "nft list ruleset", fileNameSuffix: "nftlist"},
 
 		{command: hostPrefix + "/usr/bin/" + networkPrefix + "/usr/sbin/iptables --list -v", fileNameSuffix: "iptables"},
-		{command: "ls -lsh -Z -St /dev/vfio", fileNameSuffix: "vfio-devices"},
```

Comment:
I agree that it is very annoying and disturbing when I try to understand why a job failed. Thanks for addressing this.

But isn't this sometime helpful to when you debug SRIOV / GPU clusters? Shouldn't there be an option to enable it on test runs that need it? (I really do not know.)

Reply:
Dumping info like this by its own is not helpful
```
root@sriov-worker:/# ls /dev/vfio/
100  102  104  106  108  110  112  114  116  118  120  122  124  126  128  130  132  134  136  74  76  78  80  82  84  86  88  90  92  94  96  98  vfio
101  103  105  107  109  111  113  115  117  119  121  123  125  127  129  131  133  135  73   75  77  79  81  83  85  87  89  91  93  95  97  99
```
Follow PRs can dump other info which can be meaningful upon need.

Reply:
It dumps each file's permissions and SELinux state:
```
ls -la -Z /dev/vfio
drwxr-xr-x.  2 root root system_u:object_r:device_t:s0              940 Jun 27 07:39 .
drwxr-xr-x. 15 root root system_u:object_r:container_file_t:s0     3120 Jun 27 08:27 .. 
crw-------.  1 root root system_u:object_r:vfio_device_t:s0    236,  63 Jun 23 13:45 234
...
crw-rw-rw-.  1 root root system_u:object_r:vfio_device_t:s0     10, 196 Jun 23 11:45 vfio 
```